api/api.go

```diff
@@ -117,9 +122,11 @@ func New(ctx context.Context, options *Options) (*web.API, error) {
 			filters.NewBasicAuthnFilter(options.Repository),
 			bearerAuthnFilter,
 			secfilters.NewRequiredAuthnFilter(),
-			filters.NewForbiddenLabelOperationsFilter(options.APISettings.ProctedLabels),
+			filters.NewProtectedLabelsFilter(options.APISettings.ProctedLabels),
 			&filters.SelectionCriteria{},
 			&filters.PlatformAwareVisibilityFilter{},
+			filters.NewOIDCTenantCriteriaFilter(options.APISettings.TenantCriteria),
```

Comment:
if this is to be in Peripli, it should be configurable. I don't think these 3 must be registered by default

Reply:
Its configrable with the settings - you could specify a different criteria/label to use